scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -266,31 +298,46 @@
     [(next-state state pending-job preemption-decision) preemption-decision]
     [state nil]))
 
+(defn taskless?
```

Comment:
I especially don't love my suggested name in the context it is used. 

A bit wordy but, `has-running-or-unknown-tasks?`